server/controllers/finance/reports/debtors/annual_clients_report.js

```diff
@@ -91,7 +110,8 @@ function getDebtorGroupMovements(fiscalYearId, currencyId, rate, hideLockedClien
   const hiddenClientsCondition = ' AND dg.locked = 0 ';
   const excludeCashClientsCondition = 'AND dg.is_convention = 1 ';
   const sql = `
-    SELECT ac.number AS accountNumber, dg.name AS groupName,
+    SELECT act.number AS accountNumber, dg.name AS groupName,
+      act.locked, dg.is_convention AS isConvention,
```

Comment:
In the context of this report, I believe this should be `dg.locked`, not `act.locked`.  I expected the "locked" function to be on the client, not the account.

Arguably, one could that locking a debtor group should lock their account, but that isn't how BHIMA currently (or historically) has worked.  Locking the debtor groups just prevents debtors from being billed in it, not blocking transactions to the account.  I think this makes sense on the surface (maybe you still want to be able to annul debts, but don't want any debtors to join the group or anyone else to transact on the group).

Reply:
I have fixed this and retested.